db/models/donation.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Donation = db.define('donations', {
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Sequelize.NOW
+  },
+  amount: {
+    type: Sequelize.INTEGER, // in U.S. Cents
+    allowNull: false
+  },
+  method: {
+    type: Sequelize.ENUM('stripe', 'check', 'other')
+  },
+  thanked: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  dedication: {
+    type: Sequelize.STRING
```

Comment:
Donor name, and in that case, the way in which they want to be thanked (if they do at all) (e.g. Twitter, etc.).